modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-basic-viewing-{context}"]
+= Viewing the Basic Audit Log
+
+You can view the basic audit log.
+
+.Procedure
+
+To view the basic audit log:
+
+. View the {product-title] API server logs
+
+.. If necessary, get the name of the log you want to view:
++
+----
+$ oc --insecure-skip-tls-verify adm node-logs --role=master --path=openshift-apiserver/
```

Comment:
This seems wrong for viewing something from elastic search.

Reply:
@sferich888 I am not sure this is related to ES. I think this is an API-level log function only. And not related to Elasticsearch(??). 
@jcantrill Can you comment? Is audit logging different from EFK? Is Elasticsearch required in 4.x for audit logging?